src/ui/flutter_app/lib/services/audios.dart

```diff
@@ -181,7 +181,7 @@ class Audios {
         assert(false);
       }
 
-      // TODO: isn't debug chain meant to catch errors? so why catching them in here and not in onError??
+      // TODO: [Leptopoda] isn't debug chain meant to catch errors? so why catching them in here and not in onError??
```

Comment:
I haven't looked at it in detail. You can git blame to see.

Reply:
I completely rewrote the audio back end since then. I don't think this still applies but we'll see.